config/locales/views/school_groups/school_groups.yml

```diff
@@ -185,6 +185,58 @@ en:
         <p>This page summarises these potential savings across this entire group of schools. Click on a heading to view the savings for individual schools and explore the detailed analysis.</p>
       modal_intro: This action has been identified as a priority for the following schools.
       view_analysis: View analysis
+    secr:
+      introduction: Using your half-hourly meter data, Energy Sparks is able to provide a summary of your consumption and carbon emissions to support production of your annual SECR reports.
+      section1:
+        footnotes_html: |-
+          <i>Support notes</i>
+          <p>
+          The above figures are based on the following data points. See the data downloads below for details of
+          individuals schools, meters and data quality issues.
+          </p>
+          <ul>
+          <li>Number of active schools</li>
```

Comment:
do we want numbers with these?

Reply:
Yes, I intended the supporting notes to include the numbers of each as a high level summary on coverage.